lib/vmdb/loggers.rb

```diff
@@ -116,7 +81,13 @@ def self.configure_external_loggers
       require 'inventory_refresh'
       InventoryRefresh.logger = $log
     end
-    private_class_method :configure_external_loggers
+
+    private_class_method def self.apply_plugins_config(config)
+      Vmdb::Plugins.each do |plugin|
+        logger, key = plugin.try(:logger_config_params)
```

Comment:
I wonder if it'd be better to flip this logic around.  Instead of asking the plugin for its logger and config key and setting the value in core, instruct the plugin to update its config.  This seems a little more general/pluggable and would work if a plugin had e.g. more than one logger (don't believe we have this currently but I don't see we wouldn't).

Reply:
Actually strike that, vmware has `level_vcloud` and `level_vim` so we do have that issue

Reply:
That's how I had it initially, but it pushed down logic into the plugins, which I thought was gross.  Even so, if the plugin needs a more complicated process (say chained logs, or the need to change logging of a client gem) I could see needing that.

I'll give this a shot.